core/trino-main/src/test/resources/catalog_impersonation.json

```diff
@@ -21,6 +21,16 @@
         {
             "original_user": ".*",
             "new_user": "test"
+        },
+        {
+            "original_user": "svc_(.*)",
+            "new_user": "svc_$1_prod",
+            "allow": true
+        },
+        {
+            "original_user": "external_(.*)_(.*)",
+            "new_user": "internal-$1-$2-sandbox",
```

Comment:
maybe let's replace $2 with $1 to make this example more interesting. WDYT?
```
"new_user": "internal-$2-$1-sandbox",
```

Reply:
Done.